src/query/functions/binary/binary_test.go

```diff
@@ -837,9 +837,11 @@ func TestBothSeries(t *testing.T) {
 
 			// Extract duped expected metas
 			expectedMeta := block.Metadata{Bounds: bounds}
-			expectedMeta.Tags, tt.expectedMetas = utils.DedupeMetadata(tt.expectedMetas)
+			var expectedMetas []block.SeriesMeta
```

Comment:
Can we just add a test here that has an explicit `__name__` tag please?

Reply:
Added one